packages/gestalt/src/Checkbox.js

```diff
@@ -117,7 +122,7 @@ function Checkbox({
               onFocus={() => setFocused(true)}
               onMouseEnter={handleHover}
               onMouseLeave={handleHover}
-              ref={inputElement}
+              ref={forwardedRef || inputElement}
```

Comment:
@AlbertCarreras I ran into this yesterday as well - the issue is that we're trying to combine `forwardRef` and an `innerRef`.

See https://github.com/pinterest/gestalt/blob/4fb07785648991a84e0f50bc5836dc3314b25514/packages/gestalt/src/TapArea.js#L65-L67 how we fixed it for `TapArea`.

Reply:
DONE